src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProjectOptionsProcessor.cs

```diff
@@ -36,7 +39,7 @@ internal class ProjectSystemProjectOptionsProcessor : IDisposable
     /// (especially in cases with many references).
     /// </summary>
     /// <remarks>Note: this will be null in the case that the command line is an empty array.</remarks>
-    private ITemporaryStreamStorageInternal? _commandLineStorage;
+    private TemporaryStorageHandle? _commandLineStorageHandle;
```

Comment:
note: thsi is hte one place where we dump stuff to mmfs *and* we clear out prior written info before writing out new values.